src/main/java/Task.java

```diff
@@ -0,0 +1,29 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+
+    /**
+     * Constructor for Task, generates name and status of task.
```

Comment:
Classes are well written and descriptions are given. Constructor is labelled and methods are explained